### ISSUE TITLE
cluster add-NFS mount command

### ISSUE THREAD

**Issue by ihodes** [hammerlab/stratotemplate#21]

**Comment by smondet** [comment]
Adding the NFS mount to `CLUSTER_NFS_MOUNTS` and re-doing `cluster.ml up` should just work.

Did you try that?

**Comment by ihodes** [comment]
Yes, that works; seems like that does a bunch of additional work: might it not be useful to have subcommand for just mounting a NFS or just adding packages?

**Comment by smondet** [comment]
Yes, what's tricky is keeping the "configure with just environment variables".

The CLI mount will make your `configuration.env` kinda out of sync. So the if you want to use `up` for something else (like packages) the mount won't be there (which is OK actually...).

**Comment by arahuja** [comment]
Can't up and remount be separate commands? `up` seems to reinstall a lot of packages which I've seen take a while (and has no condition so it always runs)
